packages/vitepress-plugin-git-changelog/src/vite/helpers.ts

```diff
@@ -207,3 +207,7 @@ export function parseGitLogRefsAsTags(refs?: string): string[] {
 
   return tags.map(tag => tag.replace('tag: ', '').trim())
 }
+
+export function generateCommitPathsRegExp(includeDirs: string[], includeExtensions: `.${string}`[]): RegExp {
```

Comment:
```suggestion
/**
 * Generate RegExp for filtering out paths of commits.
 *
 * It follows the rules that:
 * - include is not set, it is /^.+\md$/
 * - include is set, it is /^(${include.join('|')})\/.+\md$/
 * - in another word, /^(includeItem1|includeItem2|includeItem3)\/.+\md$/
 */
export function generateCommitPathsRegExp(includeDirs: string[], includeExtensions: `.${string}`[]): RegExp {
```

How about moving these comments here right above the function?

Reply:
Moved and modified to take into new RegExp value